go/src/meguca/server/router.go

```diff
@@ -147,6 +147,7 @@ func createRouter() http.Handler {
 		api.GET("/socket", websockets.Handler)
 		api.GET("/youtube-data/:id", youTubeData)
 		api.GET("/bitchute-title/:id", bitChuteTitle)
+		api.GET("/mod-log-entry/:type/:id", modLogEntry)
```

Comment:
We don't need a n HTTP endpoint for this functionality. All updated info should be communicated through websockets.